doc/sphinx/language/core/conversion.rst

```diff
@@ -132,7 +132,7 @@ or :term:`constants <constant>` defined in the :term:`global environment` with t
    \WFE{\Gamma}
    (x:=t:T) ∈ Γ
    --------------
-   E[Γ] ⊢ x~\triangleright_Δ~t
```

Comment:
This was introduced by #6983 (line 394 of `doc/refman/RefMan-cic.tex` used to have `\delta` for both reduction rules).